clash_lib/src/app/logging.rs

```diff
@@ -103,6 +104,11 @@ pub fn setup_logging(
         )
         .from_env_lossy();
 
+    // let trace = tracing_subscriber::filter::targets::Targets::new()
```

Comment:
you can do something like this to enable the level `RUST_LOG=h2=trace,clash=debug`